README.md

```diff
@@ -1,54 +1,65 @@
 # Elpis Next
-
-This is a slimmer rewrite of the [elpis project](https://github.com/CoEDL/elpis).
+This is a slimmer rewrite of the [elpis](https://github.com/CoEDL/elpis) project.
 
 ## Running the client and server
-
 - First, clone the repo: `git clone https://github.com/CoEDL/elpis_next.git`
 - `cd elpis_next`
 
 ### To run with docker:
-
 - `docker compose build`
 - `docker compose up`
 
-This will spawn a client service at `https://localhost:3000` and the server at `https://localhost:5001`.
-It will also create a data directory at `./data` in which to store the state of the application. This `data` directory is bound to the host machine,
-so you can inspect what's going on in the application without needing to open another terminal session inside the container.
+This will spawn a client service at `https://localhost:3000` and the server at `https://localhost:5001`. It will also create a data directory at `./data` in which to store the state of the application. This data directory is bound to the host machine, so you can inspect what's going on in the application without needing to open another terminal session inside the container.
 
-### To run a development build locally (without docker):
+#### Troubleshooting
+##### No space left on device (Docker)
+- try `docker system prune --all`
+
+If the error persists and you're trying to combine Docker Desktop with the WSL:
+- uninstall Docker Desktop
+- install Docker Engine through WSL (https://docs.docker.com/engine/install/)
 
+##### Cannot connect to the docker daemon
+- try `sudo service docker start`
+
+##### Error getting credentials (running `docker compose build`)
+Make sure you're logged in with credentials:
+- `gpg --generate-key`
+- `pass init <PUBLIC KEY (from above)>`
+- `docker login` (create login through Docker Hub if necessary)
+
+Still an issue?
+- in `~/.docker/config.json`, change `credsStore` to `credStore`
+
+##### `docker compose up` runs but doesn't set anything up at localhost:3000
+Remove these lines from `./compose.yaml`:
```

Comment:
This is really interesting and should be made a github issue I reckon. Is this a windows dependent thing do you know?

Reply:
https://blog.adamgamboa.dev/docker-fixing-error-storing-credentials/
looks like it's a case of different versions of docker using different names